nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/AbstractExecuteSQL.java

```diff
@@ -94,6 +106,16 @@
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
             .build();
 
+    public static final PropertyDescriptor SQL_POST_QUERY = new PropertyDescriptor.Builder()
+            .name("sql-post-query")
+            .displayName("SQL Post-Query")
+            .description("SQL post-query to execute. Semicolon-delimited list of queries. "
```

Comment:
Likewise, this description is duplicative with the name. "Queries that will be executed after the intended query."

Reply:
"A semicolon-delimited list of queries executed after the main SQL query is executed. Results/outputs from these queries will be suppressed if there are no errors."